.changeset/empty-chairs-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+"thirdweb": minor
```

Comment:
does this add any new APIs to the public API? - If so `minor` is appropriate, if not this should be `patch` (if it's only internals)

Reply:
If we have the new account type yes, otherwise no. Will update based on what we decide to do there.

Reply:
makes sense